prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1179,18 +1179,16 @@ int rxr_ep_init(struct rxr_ep *ep)
 		goto err_free_tx_pool;
```

Comment:
Please fix this commit so it's authored by you.

Reply:
Thanks. will fix